sdk/eventhub/azure-eventhubs/HISTORY.rst

```diff
@@ -3,7 +3,7 @@
 Release History
 ===============
 
-1.3.3 (2019-11-20)
+1.3.3 (2019-12-4)
 ------------------
```

Comment:
We should add the changes to the release notes:
```
**Features**
- Added `idle_timeout` parameter to the `add_receiver` functions and `EPHOptions` for optionally closing idle connections.
- Added `reconnect_timeout` and `max_reconnect_tries` parameters to `receive` functions for better control of connection behaviour during receive.

**Bug Fixes**
- Fixed bug where on reconnect, `receive` functions returned an empty list.
- Fixed bug in partition pump logger.
```

Reply:
If I recall correctly, the application properties fix also needs to be added to the release notes.

Reply:
I think this fix will also be in the next release?
https://github.com/Azure/azure-sdk-for-python/issues/8227

Any others you can think of?

Reply:
properties fix should already be detailed via "- Fixes bug preventing application_properties from being transmitted when set individually in key-value form."  but the others are ++added, thanks as always!  The only other thing I can think of is the "send timeout bugfix" that was half UAMQP and half here, I'll make a note.